docs/api/_index.md

```diff
@@ -25,6 +25,8 @@ For the sake of clarity, in this document we have grouped API endpoints by servi
 | [Remote write](#remote-write) | Distributor | `POST /api/v1/push` |
 | [Tenants stats](#tenants-stats) | Distributor | `GET /distributor/all_user_stats` |
 | [HA tracker status](#ha-tracker-status) | Distributor | `GET /distributor/ha_tracker` |
+| [Pprof](#pprof) | Debugger | `GET /debug/pprof` |
+| [Fgprof](#fgprof) | Debugger | `GET /debug/fgprof` |
```

Comment:
These are exposed by all Cortex services:

```suggestion
| [Pprof](#pprof) | _All services_ | `GET /debug/pprof` |
| [Fgprof](#fgprof) | _All services_ | `GET /debug/fgprof` |
```

Reply:
Because of this, please move it above, at the end of the "All services" group.

Reply:
Moved